Client/src/Components/Game/Users/PlayersList/PlayersList.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+	import type Player from 'Models/Game/Player';
+
+	import UsePlayersList from './usePlayersList';
+	import PlayerCard from './Player/PlayerCard.svelte';
+
+	let players: Player[] = [];
+	const { getPlayers } = UsePlayersList();
+
+	onMount(async () => {
+		players = await getPlayers();
+	});
+</script>
+
+<div class="grid grid-cols-1">
+	{#each players as player}
+		<div class="col-span-1">
+			<PlayerCard player="{player}" />
```

Comment:
I would say "PlayerRowInCard" or "PlayerListItem", what do you think?

Reply:
Yeah, it's not a card

Reply:
Maybe PlayerRow

Reply:
PlayerRow is also nice, if you want to keep it short